aioredisqueue/queue.py

```diff
@@ -6,6 +6,7 @@
 class Queue(object):
 
     def __init__(self, redis, key_prefix='aioredisqueue_', *,
+                 serialization_method=None,
```

Comment:
* let's add ability to use `dynamic` method.

Reply:
let's set default for `key_prefix` to None: and set it dynamically to if this param is None to 'aioredisqueue_' + serialization_method